### PR TITLE
Add snpeff

### PR DESCRIPTION
**Pull request by moonso**
- Can use SnpEff annotation
- Updated Transcript model
- Updated requirements

### CODE REVIEW

**Comment by moonso** [comment]
@robinandeer please have a look when you have a minute

**Comment by moonso** [comment]
ping @brainstorm @guillermo-carrasco

File: puzzle/utils/get_info.py [hunk 1]
```diff
@@ -72,7 +74,7 @@ def get_most_severe_consequence(transcripts):
     most_severe_score = None
 
     for transcript in transcripts:
-        for consequence in transcript['Consequence'].split('&'):
```
Comment (robinandeer) [inline puzzle/utils/get_info.py]: must be changed in the fixtures

**Comment by robinandeer** [comment]
- [ ] Change link for build badge: https://travis-ci.org/robinandeer/puzzle.svg?branch=master
- [ ] Remove link to demo in README

**Pull request merged**